redwood-runtime/src/commonMain/kotlin/app/cash/redwood/LayoutModifier.kt

```diff
@@ -111,6 +119,11 @@ public class CombinedLayoutModifier(
   override fun <R> foldOut(initial: R, operation: (LayoutModifier.Element, R) -> R): R =
     outer.foldOut(inner.foldOut(initial, operation), operation)
 
+  override fun forEach(block: (LayoutModifier.Element) -> Unit) {
+    outer.forEach(block)
+    inner.forEach(block)
+  }
```

Comment:
We should probably start adding tests for these. There's probably a bunch for existing functions that I never imported when I pulled in the type, too.

Reply:
The only test I can find is https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/ui/ui/src/test/kotlin/androidx/compose/ui/ModifierTest.kt;l=22?q=ModifierTest&sq=&ss=androidx%2Fplatform%2Fframeworks%2Fsupport and it's for `foldIn` and `foldOut` which I suspect we'll be deleting once this merges and the generated serialization code is migrated to `forEach`. So, yeah. Feel free to just test your new changes.

Reply:
Sounds good. `forEach` isn't present in Compose UI, but if we're OK with diverging from Compose UI's `Modifier` then I'll remove `foldIn` and `foldOut` in a subsequent PR + add tests.